packages/core/package.json

```diff
@@ -53,7 +53,8 @@
     "react-helmet": "5.2.1",
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
-    "react-sparklines": "^1.7.0"
+    "react-sparklines": "^1.7.0",
+    "react-use": "^14.1.1"
```

Comment:
We're currently on v13, do you need the version bump or is it alright to stick to v13 for now and we'll bump everything later?

Reply:
Bumped since this comment